### PR TITLE
[SLA] PIM-5478: Change string to array for user_groups restriction in attribute searchable repository

### PR DESCRIPTION
**Pull request by pierallard**
| Q                 | A
| ----------------- | ---
| Specs             | n
| Behats            | y
| Blue CI           | processing
| Changelog updated | y
| Review and 2 GTM  | TODO

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

File: CHANGELOG-1.4.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.4.x
+
+## Bug fixes
+- PIM-5478: Change string to array for user_groups restriction in attribute searchable repository
```
Comment (solivier) [inline CHANGELOG-1.4.md]: it does not explain the bug

**Comment by solivier** [comment]
GTM after comments fixed

**Comment by phaseinducer** [comment]
no specs for the repo ?

**Comment by pierallard** [comment]
@phaseinducer we don't spec the repo

**Pull request merged**